jobs/env/ci-kubernetes-e2e-kops-aws-sig-cli.env

```diff
@@ -0,0 +1 @@
+KOPS_PUBLISH_GREEN_PATH=gs://kops-ci/bin/latest-ci-green.txt
```

Comment:
you don't want to overwrite the last-ci-green, I think just get rid of it

Reply:
I just copied it directly from jobs/env/ci-kubernetes-e2e-kops-aws.env so the environment would be the same but I think you're right that I should just get rid of it